### PR TITLE
[FIX] Fix beam search test

### PR DESCRIPTION
**Pull request by zhuohan123**
Revert a change in #2869 that causes beam search test to fail.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -13,10 +13,12 @@
 _LONG_PROMPTS = [os.path.join(_TEST_DIR, "prompts", "summary.txt")]
 
 
-def _read_prompts(filename: str) -> List[str]:
+def _read_prompts(filename: str) -> str:
```
Comment (WoosukKwon) [inline tests/conftest.py:16]: IIUC
```suggestion
def _read_prompts(filename: str) -> List[str]:
```

**Review by WoosukKwon — APPROVED** [review]
LGTM. Thanks for the fix.

**Comment by ronensc** [comment]
Please note that the proposed change currently only reads the first line (i.e., the first prompt) of the file and ignores the others. 

AFAIU, the beam search test fails on the second prompt. I believe the fix should be applied to the beam search test. A temporary solution would be to pass only the first prompt to the beam search test. This way, all other tests relying on `_read_prompts()` would benefit from testing with multiple prompts.

**Comment by zhuohan123** [comment]
> Please note that the proposed change currently only reads the first line (i.e., the first prompt) of the file and ignores the others.
> 
> AFAIU, the beam search test fails on the second prompt. I believe the fix should be applied to the beam search test. A temporary solution would be to pass only the first prompt to the beam search test. This way, all other tests relying on `_read_prompts()` would benefit from testing with multiple prompts.

Thanks for pointing this out! Changed this PR accordingly.

**Comment by WoosukKwon** [comment]
@zhuohan123 Let's merge it?

**Pull request merged**